lib/active_job/queue_adapters/solid_queue_ext/workers.rb

```diff
@@ -4,13 +4,13 @@ def exposes_workers?
   end
 
   def fetch_workers(workers_relation)
-    solid_queue_processes_from_workers_relation(workers_relation).collect do |process|
+    workers(workers_relation).collect do |process|
```

Comment:
Why the name change?

Reply:
just naming styling, I think I'd go with the old one tough